integration-tests/plugins/index.js

```diff
@@ -40,5 +40,7 @@ module.exports = on => {
     stubFindUsers: auth.stubFindUsers,
 
     stubUnverifiedUserDetailsRetrieval: auth.stubUnverifiedUserDetailsRetrieval,
+
+    stubVerifyToken: (active = true) => auth.stubVerifyToken(active),
```

Comment:
`stubVerifyToken: auth.stubVerifyToken,` ? so you don't need to do the defaulting in two places?

Reply:
correct